docs/modules/storage/pages/configuring-persistence.adoc

```diff
@@ -1261,6 +1261,55 @@ config.addMapConfig(mapConfig);
 ====
 |===
 
+[[sql]]
+== SQL Metadata Options
+
+Use the following configuration to enable the cluster-wide persistence of SQL metadata to disk. With persistence enabled, SQL mappings, data connections, and views are retained after the cluster restarts.
+
+[tabs] 
+==== 
+XML:: 
++ 
+--
+[source,xml]
+----
+<hazelcast>
+  <map name="__sql.catalog">
+    <data-persistence enabled="true"/>
+  </map>
+</hazelcast>
+----
+--
+YAML:: 
++ 
+--
+[source,yaml]
+----
+hazelcast:
+  map:
+  __sql.catalog:
+    data-persistence:
+      enabled: true
+----
+--
+Java:: 
++ 
+--
+[source,java]
+----
+Config config = new Config();
+
+MapConfig mapConfig = config.getMapConfig("__sql.catalog");
+mapConfig.getDataPersistenceConfig().setEnabled(true);
+mapConfig.getSqlConfig().setEnabled(true);
+
+config.addMapConfig(mapConfig);
+----
+--
+====
+
+NOTE: When implementing permissions for the SQL catalog IMap (`__sql.catalog`), we do not recommend using wildcards (*).  
```

Comment:
We probably want to get rid of this line altogether given that we don't want users to configure anything about `__sql.catalog` directly (I'm not sure it includes security specifically though).

Reply:
Removed.